Dockerfile

```
@@ -12,7 +12,7 @@ FROM   ubuntu:18.04
 ENV    DEBIAN_FRONTEND noninteractive
 
 # Download and install everything from the repos.
-RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common
+RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common sudo
```

As mentioned, can you remove `sudo` here

```suggestion
RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common
```

---

I managed to screw this up, so I closed the PR. Will fix it locally, and submit a new PR. :)

---

No problem, I'm happy to review your next PR :)